tests/api/api-client.ts

```diff
@@ -59,4 +60,81 @@ export class ApiClient {
 
     return responseBody.id
   }
+
+  //homework12
+
+  async createOrderAndRetrieveOrderId(): Promise<number> {
+    console.log('Creating order...')
+    const response = await this.request.post(`${serviceURL}${orderPath}`, {
+      data: OrderDto.createOrderWithUndefinedOrderId(),
+      headers: {
+        Authorization: `Bearer ${this.jwt}`,
+      },
+    })
+    console.log('Order response: ', response)
+
+    expect(response.status()).toBe(StatusCodes.OK)
+    const responseBody = await response.json()
+
+    const orderId = responseBody.id
+    expect.soft(orderId).toBeDefined()
+
+    console.log('Order retrieval with ID :${id}')
+    const getOrderResponse = await this.request.get(`${serviceURL}${orderByIdPath(orderId)}`, {
+      headers: {
+        Authorization: `Bearer ${this.jwt}`,
+      },
+    })
+    console.log('Order retrieval response: ', getOrderResponse)
+    expect(getOrderResponse.status()).toBe(StatusCodes.OK)
+
+    const getOrderResponseBody = await getOrderResponse.json()
+    console.log('Order retrieved: ', getOrderResponseBody)
+
+    return getOrderResponseBody
+  }
+
+  async createOrderAndDeleteOrderId(): Promise<number> {
```

Comment:
you already have 'createOrder' implementation so don;t need to put it here

just add actions for  retreive and delete to api client

Reply:
Fixed api-client, but also need to fix at test in auth-order-creation.spec.ts